src/PuppetLanguageClient.ts

```diff
@@ -2,7 +2,8 @@ import { ConnectionStatus } from "./interfaces";
 import { LanguageClient } from "vscode-languageclient/lib/main";
 import { PuppetStatusBar } from "./PuppetStatusBar";
 import { ILogger } from "./logging";
-import { ServerOptions, LanguageClientOptions } from "vscode-languageclient/lib/client";
+import { LanguageClientOptions } from "vscode-languageclient/lib/client";
```

Comment:
Nit: this can be one line if you remove lib/client. This can be done in a new PR